plugins/frontend/backstage-plugin-github-pull-requests/src/api/GithubPullRequestsClient.ts

```diff
@@ -36,28 +39,37 @@ export class GithubPullRequestsClient implements GithubPullRequestsApi {
     page?: number;
     state?: PullRequestState;
     baseUrl: string | undefined;
+    etag: string
   }): Promise<{
     maxTotalItems?: number;
     pullRequestsData: PullsListResponseData;
+    etag?: string
```

Comment:
Is this not returned from Octokit every time?

Reply:
Oh, you are right, for pullrequests it is returned. The api does not return for every endpoint the etag, but in this case all I care is the pullrequests. Ty!